client/modules/workspace/src/SelectModal/SelectModal.tsx

```diff
@@ -209,78 +294,109 @@ export const SelectModal: React.FC<{
-                      path={selectedPath}
-                      columns={FileColumns}
-                      rowSelection={undefined}
-                      scheme={scheme}
-                      scroll={undefined}
-                    />
-                  </div>
-                </>
-              )}
-              {showProjects && (
-                <div className={styles.destFilesTableContainer}>
-                  <SelectModalProjectListing
-                    onSelect={(uuid, projectId) =>
-                      onProjectSelect(uuid, projectId)
-                    }
-                  />
-                </div>
-              )}
-            </section>
+    <Modal
+      open={isModalOpen}
+      onCancel={handleClose}
+      width="40%"
+      footer={null}
+      styles={modalStyles}
+      title={<span className={styles.modalTitle}>Select</span>}
+    >
+      <article className={styles.modalContent}>
+        <div className={styles.modalPanel}>
+          <div className={styles.dataFilesModalColHeader}>Select Input</div>
+          <div className={styles.selectRowContainer}>
+            <div className={styles.systemSelectRow}>
+              <ConfigProvider theme={systemSelectThemeConfig}>
+                <Select
+                  options={systemOptions}
+                  virtual={false}
+                  value={dropdownValue}
+                  onChange={(newValue) => dropdownCallback(newValue)}
+                  className={styles.systemSelect}
+                  popupMatchSelectWidth={false}
+                  suffixIcon={<SystemSuffixIcon />}
+                />
+              </ConfigProvider>
+            </div>
+            <div className={styles.selectRowItem}>
+              <BaseFileListingBreadcrumb
+                className={styles.breadCrumbItem}
+                api={selectedApi}
+                system={selectedSystem}
+                path={decodeURIComponent(selectedPath)}
+                systemRootAlias={selection.projectId}
+                initialBreadcrumbs={
+                  selectedSystem.startsWith('project-')
+                    ? [{ title: 'My Projects', path: 'PROJECT_LISTING' }]
+                    : []
+                }
+                itemRender={(item) => {
+                  return (
+                    <Button
+                      type="link"
+                      onClick={() => navCallback(item.path ?? '')}
+                    >
+                      {item.title}
+                    </Button>
+                  );
+                }}
+              />
+            </div>
           </div>
-        </article>
-      </Modal>
-    </>
+          <Form
+            form={form}
+            onFinish={(data) => {
+              if (data.query) {
+                setSearchTerm(data.query);
+              } else {
+                setSearchTerm(null);
+              }
+            }}
+            style={{ display: 'inline-flex', marginBottom: '10px' }}
```

Comment:
could maybe bump this marginBottom to 15 or 20px to match design

Reply:
done